### PR TITLE
Java Feb 20 Release Notes

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: releases/2020-02/java.md [hunk 1]
```diff
@@ -128,27 +122,48 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
+
+### Storage Blob Cryptography ([Changelog]())
+
+### Storage Files Datalake ([Changelog]())
+
+### Storage Files Shares ([Changelog]())
+
+### Storage Queues ([Changelog]())
+
+### Text Analytics ([Changelog]())
```
Comment (mssfang) [inline releases/2020-02/java.md]:
```suggestion
### Text Analytics ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md#100-beta2-2020-02-12))
```

File: releases/2020-02/java.md [hunk 2]
```diff
@@ -128,27 +122,48 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
+
```
Comment (mssfang) [inline releases/2020-02/java.md]:
```suggestion
- Added the ability to create a Text Analytics client with a credential that can be updated in long-lived clients.
- All batch overload methods have been renamed by adding the suffix `Batch`.
- All single text operation methods now return an atomic type of operation result.
```

File: releases/2020-02/java.md [hunk 3]
```diff
@@ -74,43 +74,37 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
-
-<dependency>
-  <groupId>com.azure</groupId>
-  <artifactId>azure-storage-queue</artifactId>
   <version>12.2.0</version>
 </dependency>
 
 <dependency>
   <groupId>com.azure</groupId>
-  <artifactId>azure-core-tracing-opencensus</artifactId>
```
Comment (alzimmermsft) [inline releases/2020-02/java.md]: Do we want to include modules we are no longer supporting? OpenCensus is no longer under active development from my understanding and has been superseded by OpenTelemetry.

Comment (samvaity) [reply releases/2020-02/java.md]: Yes, we no longer support opencensus. This can be removed.

Comment (joshfree) [reply releases/2020-02/java.md]: Let's remove opencensus, and keep opentelemetry, just so the latest release notes always list the latest set of packages which work together

**Pull request merged**